2-structured-data/books/model-datastore.js

```diff
@@ -18,7 +18,7 @@ const config = require('../config');
 
 // [START config]
 const ds = Datastore({
-  projectId: config.get('GCLOUD_PROJECT'),
+  projectId: config.get('GOOGLE_CLOUD_PROJECT'),
```

Comment:
Why do you need to set the project ID here?
The library is supposed to take it from the metadata server or from the GOOGLE_APPLICATION_CREDENTIAL context